Resources/Prototypes/DeadSpace/Entities/Clothing/Masks/masks.yml

```
@@ -13,3 +13,24 @@
     tags:
     - WhitelistChameleon
   - type: IngestionBlocker
+
+- type: entity
+  parent: ClothingMaskPullableBase
+  id: ClothingMaskFakeWizHatBeard
+  name: мудрая борода
+  description: Для пожилых и мудрых существ, на 100% состоит из синтепона.
+  components:
+  - type: Sprite
+    sprite: DeadSpace/Clothing/Head/Mask/FakeWizHat.rsi
+  - type: Clothing
+    sprite: DeadSpace/Clothing/Head/Mask/FakeWizHat.rsi
```

```suggestion
  - type: Sprite
    sprite: DeadSpace/Clothing/Head/Mask/wizard_fake_beard.rsi
  - type: Clothing
    sprite: DeadSpace/Clothing/Head/Mask/wizard_fake_beard.rsi
```
И соответственно так же измени папку